src/System.Runtime.Serialization.Xml/tests/S.P.DCS.Tests.cs

```diff
@@ -0,0 +1,93 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Xml;
+using Xunit;
+
+public static class SystemPrivateDataContractSerializationTest
```

Comment:
If the tests are specific to the implementation, then they should be moved to System.Private.DataContractSerialization solution. Otherwise, we should avoid using names that suggests they're tied to implementation.